greeclimate/device.py

```diff
@@ -105,7 +105,11 @@ def generate_temperature_record(temp_f):
 TEMP_OFFSET = 40
 TEMP_MIN_F = 46
 TEMP_MAX_F = 86
-TEMP_TABLE = [generate_temperature_record(x) for x in range(TEMP_MIN_F, TEMP_MAX_F + 1)]
+TEMP_MIN_TABLE = 8
```

Comment:
Since the goal here is to allow the device to report temperatures outside of the settable range, MIN should also be lowered.... maybe -50, then max to 60... just to give some headroom of conceivable reasonable values.

Reply:
I understand your point, but if the unit inside a room is reading less than 8C, or 46F, there's a bigger issue at hand than the temperature it's reading. Fair point, should be fine to reduce to 0C.

Reply:
Extended the minimum to -60 C (-76 F), and maximum to 60 C (140 F).